osp/models/question.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+from osp.models.abstract_timestamp import AbstractTimestamp
+
+SA = 'char'
+PA = 'text'
+CF = 'choice'
+CH = 'checkbox'
+DD = 'dropdown'
+FI = 'file'
+DA = 'date'
+TI = 'time'
+DATA_TYPE = (
+    (SA, 'Short Answer'),
+    (PA, 'Paragraph Answer'),
+    (CF, 'Choice'),
+    (CH, 'Checkbox'),
+    (DD, 'Dropdown'),
+    (FI, 'File Upload'),
+    (DA, 'Date'),
+    (TI, 'Time')
+)
+
+class Question(AbstractTimestamp):
+
+    label = models.CharField(max_length=255)
+    data_type = models.CharField(
+        max_length=8,
```

Comment:
Should be 16, `Paragraph Answer` is the longest

Reply:
It is taking the values of checkbox and dropdown to be the longest, that is, 8. Received this error in migrations, and was asked to keep maximum to be 8.